scripts/gendocs/generate

```diff
@@ -167,9 +180,6 @@ rm -r pkg/apis/certmanager/v1alpha3
 rm -r pkg/apis/certmanager/v1beta1
 gendocs "v1.6-docs"
 
-# when adding a new version here run
-# ./scripts/gendocs/generate 1 to force generation of CLI docs
-# and add them to the git index
```

Comment:
This comment was already updated by @jahrlin  but is now removed because the new pattern is established in L149-150 above